src/XPurchase/VirtoCommerce.XPurchase/Extensions/FieldBuilderHelper.cs

```diff
@@ -0,0 +1,45 @@
+using GraphQL;
+using GraphQL.Builders;
+using GraphQL.Types;
+using MediatR;
+using VirtoCommerce.ExperienceApiModule.Core.Infrastructure;
+using VirtoCommerce.XPurchase.Commands;
+using VirtoCommerce.XPurchase.Schemas;
+
+namespace VirtoCommerce.XPurchase.Extensions
+{
+    public static class FieldBuilderHelper
+    {
+        public static EventStreamFieldType CreateCartAggregateMutation<TArgumentType, TCartCommand>(string mutationName, IMediator mediator)
```

Comment:
I think these methods should be generalized (which could be too complex) or be just private methods in Purchase schema. Now they include too much Cart specific, I do not see them as generic FieldBuilderHelper methods.